commands.md

```diff
@@ -8,6 +8,7 @@ Command | Implemented By | Who can run it | Description
 `/unassign [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Unassigns specified people (or yourself if no one is specified). Target must already be assigned.
 `/cc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Request review from specified people (or yourself if no one is specified). Target must be a kubernetes org member.
 `/uncc [@userA @userB @etc]` | prow [assign](./prow/plugins/assign) | anyone | Dismiss review request for specified people (or yourself if no one is specified). Target must already have had a review requested.
+`/help` | prow [help](./prow/plugins/help) | anyone | If issue is unassigned, adds help-wanted label. Label is removed when issue is assigned.
```

Comment:
We now have [3270 open issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee) without assignee. Do we really want to label all these issues as `help-wanted`? I'm not sure that all of them are indeed wanting help. Some of them are umbrella issue or just for tracking something.